cmd/create/cluster/cmd.go

```diff
@@ -1090,8 +1092,9 @@ func run(cmd *cobra.Command, _ []string) {
 					billingAccount = aws.ParseOption(billingAccount)
 				}
 
-				if billingAccount == "" || !ocm.IsValidAWSAccount(billingAccount) {
```

Comment:
In the function `validateBillingAccount` the condition is different, is that expected?

Reply:
Nice catch! Test case should probably cover the empty case as well.

Reply:
Nice catch @oriAdler